docs/install/releases/feature-stages.md

```diff
@@ -1,45 +1,64 @@
+
+This gives Coder users the opportunity to use the latest Coder features and improve the open-source project.
 
 If you encounter an issue with any Coder feature, please submit a
 [GitHub issue](https://github.com/coder/coder/issues) or join the
 [Coder Discord](https://discord.gg/coder).
 
 ## Feature stages
 
-| Feature stage                          | Stable | Production-ready | Support               | Description                                                                                                                   |
-|----------------------------------------|--------|------------------|-----------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| [Early Access](#early-access-features) | No     | No               | GitHub issues         | For staging only. Not feature-complete or stable. Disabled by default.                                                        |
-| [Beta](#beta)                          | No     | Not fully        | Docs, Discord, GitHub | Publicly available. In active development with minor bugs. Suitable for staging; optional for production. Not covered by SLA. |
-| [GA](#general-availability-ga)         | Yes    | Yes              | License-based         | Stable and tested. Enabled by default. Fully documented. Support based on license.                                            |
+| Feature stage                             | Stable | Production-ready | Support                                 | Description                                                                                                                                                                             |
+|-------------------------------------------|--------|------------------|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Experiment](#experiment-hidden)          | No     | No               | N/A                                     | For staging and testing only. Not feature-complete or stable.</br>Hidden and disabled by default.                                                                                       |
+| [Early Access](#early-access-features)    | No     | No               | Limited documentation</br>GitHub issues | For staging only. Functional, but not feature-complete or stable.</br>Disabled by default.                                                                                              |
+| [Beta](#beta)                             | No     | Not fully        | Documentation</br>Discord</br>GitHub    | Publicly available on an opt-in basis. In active development with minor bugs. Suitable for staging; optional for production. Not covered by SLA.                                        |
```

Comment:
Another product question: will all `beta` features be opt-in? Or will some just be tagged `beta`?
What if something is greenfield?